### ISSUE TITLE
Cast exception on empty header values

### ISSUE THREAD

**Issue by WebDucer** [nmklotas/GitLabApiClient#74]
I get an cast exception on empty header values for next page. I use GitLab 12.3.5. If I try the request in Postman, I see, that `X-Prev-Page` is empty. `GetFirstHeaderValueOrDefault` checks only for `null`. The same for `X-Next-Page`, if there a only one page of data.

I will create a PR with fix for this.

```
System.FormatException : Input string was not in a correct format.
   at System.Number.ThrowOverflowOrFormatException(ParsingStatus status, TypeCode type)
   at System.String.System.IConvertible.ToInt32(IFormatProvider provider)
   at System.Convert.ChangeType(Object value, Type conversionType, IFormatProvider provider)
   at System.Convert.ChangeType(Object value, Type conversionType)
   at GitLabApiClient.Internal.Http.HttpResponseHeadersExtensions.GetFirstHeaderValueOrDefault[T](HttpResponseHeaders headers, String headerKey) in C:\src\External\GitLabApiClient\src\GitLabApiClient\Internal\Http\GitLabApiPagedRequestor.cs:line 110
   at GitLabApiClient.Internal.Http.GitLabApiPagedRequestor.GetPagedList[T](String url) in C:\src\External\GitLabApiClient\src\GitLabApiClient\Internal\Http\GitLabApiPagedRequestor.cs:line 27
```

**Comment by casz** [comment]
My bad I should have added tests 😰

**Comment by dragongun200** [comment]
same Problem here

**Comment by casz** [comment]
fixed in #75 released in https://github.com/nmklotas/GitLabApiClient/releases/tag/1.1.11
